docs/cli/start.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "wash start"
+draft: false
+sidebar_position: 23
+description: "wash start command reference"
+--- 
+
+`wash start` assists in launching an actor or a provider from an OCI reference on a host. The user may specify if they want to start an actor or a provider and pass the OCI reference of that entity to the command. By default, the actor or provider are auctioned in the lattice for a suitable host. To specify a host to launch the actor or provider, pass the host ID using the `--host-id` flag. Following are the available subcommads:
```

Comment:
We should also document the ability to use a file with the `file://` prefix as well as the requirement for starting the host with the right environment variables to allow it